packages/core-utils/test/common/hex-utils.spec.ts

```diff
@@ -18,3 +19,24 @@ describe('getRandomHexString', () => {
     expect(getRandomHexString(8)).to.equal('0x' + '88'.repeat(8))
   })
 })
+
+describe('toRpcHexString', () => {
+  it('should parse 0', () => {
+    expect(toRpcHexString(0)).to.deep.equal('0x0')
+    expect(toRpcHexString(BigNumber.from(0))).to.deep.equal('0x0')
+  })
+
+  it('should parse non 0', () => {
+    const cases = [
+      { input: 2, output: '0x2' },
+      { input: BigNumber.from(2), output: '0x2' },
+      { input: 100, output: '0x64' },
+      { input: BigNumber.from(100), output: '0x64' },
+      { input: 300, output: '0x12c' },
+      { input: BigNumber.from(300), output: '0x12c' },
+    ]
+    for (const test of cases) {
+      expect(toRpcHexString(test.input)).to.deep.equal(test.output)
+    }
+  })
```

Comment:
You should be able to use `it.each` here

Reply:
That isn't built into mocha to my knowledge. Would require adding another dependency. If it can run in parallel that would be nice

Reply:
Oh I see. I'm used to it from Jest.